src/eval.cpp

```diff
@@ -727,14 +727,14 @@ std::string func2call(int func)
 	return s;
 }
 
-WELL512 random;
+WELL512 wrandom;
```

Comment:
this probably should be gRandom, my bad

Reply:
I can change that, no worries.
It is just that we get a conflict with `random` function, leading to a build failure